thrift/atoms/tenfour.thrift

```diff
@@ -0,0 +1,61 @@
+namespace * tenfour
+
+include "../shared.thrift"
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+struct Content {
```

Comment:
It's namespaced to `tenfour` though if that makes sense. However, we can of course change it. Currently I have tried to keep it the same as the model in the tenfour repo.